yoti_python_sdk/sandbox/client.py

```diff
@@ -0,0 +1,184 @@
+from yoti_python_sdk.sandbox.endpoint import SandboxEndpoint
+from cryptography.fernet import base64
+from os.path import expanduser, isfile
+from past.builtins import basestring
+
+import json
+import requests
+import yoti_python_sdk
+from yoti_python_sdk.config import (
+    X_YOTI_AUTH_KEY,
+    X_YOTI_AUTH_DIGEST,
+    X_YOTI_SDK,
+    SDK_IDENTIFIER,
+    X_YOTI_SDK_VERSION,
+    JSON_CONTENT_TYPE,
+)
+from yoti_python_sdk.sandbox.token import YotiTokenRequest, YotiTokenResponse
+from yoti_python_sdk.sandbox.attribute import SandboxAttribute
+from yoti_python_sdk.sandbox.anchor import SandboxAnchor
+from yoti_python_sdk.crypto import Crypto
+from json import JSONEncoder
+
+
+class SandboxEncoder(JSONEncoder):
+    def default(self, o):
+        if isinstance(o, YotiTokenRequest):
+            return o.__dict__()
+        if isinstance(o, SandboxAttribute):
+            return o.__dict__()
+        if isinstance(o, SandboxAnchor):
+            return o.__dict__()
+
+        return json.JSONEncoder.default(self, o)
+
+
+class SandboxClient(object):
+
```

Comment:
We shouldn't have a blank line after class declarations, guessing this isn't picked up by any of our pre-commit checks?
```suggestion
```